### ISSUE TITLE
Contribute to Ignite

### ISSUE THREAD

**Issue by sirinath** [sumeet70/aiex#3]
Wondering if you are willing to contribute this to the Ignite project?

**Comment by sumeet70** [comment]
Yes please. That was the original intent. Wanted to take it further, but
then life happened.
